### DISCUSSION TITLE
加载不出来

### DISCUSSION THREAD

**Discussion by licalleni** [share121/inter-knot#179] [其他]
test

**Comment by share121** [comment]
正在修复

**Comment by share121** [comment]
修好了

**Reply by licalleni** [reply]
不好意思，是我的浏览器版本太低了
![ab4b8062ly1hqk3vze76dj20u00ss409](https://github.com/user-attachments/assets/f3d635d3-8cc0-4cc3-98ef-a93a5e9c5255)

**Comment by NOVAQUQ** [comment]
![img_-981412852_1690387176972](https://github.com/user-attachments/assets/f6cb87e6-9b78-411f-9ed1-ae75542ae9bb)
好的谢谢，你真可爱∠( ᐛ 」∠)_
